repair_line_sequence/__manifest__.py

```
@@ -0,0 +1,18 @@
+    "version": "17.0.1.0.0",
+    "website": "https://github.com/OCA/repair",
+    "author": "Cetmix, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "depends": ["repair"],
+    "data": [
+        "views/repair_order_views.xml",
+    ],
+    "post_init_hook": "post_init_hook",
+    "installable": True,
+    "application": False,
+}
```

Pylint (3.3.7)

[warning] Statement seems to have no effect

(W0104)